<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTable</class>
 <widget class="QWidget" name="FormTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>54</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTeaBuild1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTeaBuild2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTeaBuild3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExpBuild1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExpBuild2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExpBuild3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>235</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultHall">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQiuShi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>325</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShareSqr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDutyRoom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>415</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAll">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
